.github/workflows/ci.yml

```diff
@@ -0,0 +1,14 @@
+on: [push, pull_request]
+
+name: CI
+
+jobs:
+  build_and_test:
+    name: Rust project
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions-rs/toolchain@v1
+        with:
+          toolchain: nightly
+      - run: cargo build
```

Comment:
Shouldn't it be cargo test?

Reply:
The goal is to have it in two steps so you can easily identify whether just test failed, or just build failed

Reply:
```suggestion
```